packages/electrod/electrod.0.2.2/opam

```diff
@@ -0,0 +1,48 @@
+opam-version: "2.0"
+maintainer: "David Chemouil <david.chemouil+git@onera.fr>"
+authors: ["David Chemouil" "Julien Brunel"]
+homepage: "https://grayswandyr.github.io/electrod"
+bug-reports: "https://github.com/grayswandyr/electrod/issues"
+license: "MPL-2.0"
+dev-repo: "git+https://github.com/grayswandyr/electrod.git"
+doc: "https://grayswandyr.github.io/electrod/api"
+build: [
+  ["dune" "subst"] {pinned}
+  ["dune" "build" "-p" name "-j" jobs]
+]
+depends: [
+  "ocaml" {>= "4.04"}
+  "dune" {>= "1.4"}
+  "cmdliner" 
+  "containers" {>= "2.0"}
+  "fmt"
```

Comment:
```suggestion
  "fmt" {>= "0.8.7"}
```
I'm guessing the error I showed you above comes from this missing constraint (just guessing, please double-check)